docs/howto/operator.rst

```diff
@@ -102,6 +102,62 @@ to execute a BigQuery load job.
     :start-after: [START howto_operator_gcs_to_bq]
     :end-before: [END howto_operator_gcs_to_bq]
 
+GceInstanceStartOperator
```

Comment:
It looks like it's the right time to split the GCP operator into separate chapters for each "entity" (GCE/GCF/BigQuery)

Reply:
Yeah. I think it makes sense to split it per "service". But let's just start discussion about that in the GCP Service Airflow operators guide? Can you do that ?

Reply:
After offline discussion with @potiuk: it has already been done by Jarek.